tests/Common/DevBranchesTest.php

```diff
@@ -290,6 +294,44 @@ public function testReadOnlyRoleBranchesPermissions()
         $this->assertCount(2, $adminDevBranches->listBranches());
     }
 
+    public function testCreateAndDeleteBranchWithWorkspace()
+    {
+        $this->_client->verifyToken();
+        $branchesApi = new DevBranches($this->_client);
+
+        $branchName = $this->generateBranchNameForParallelTest();
+        $branchDescription = $this->generateDescriptionForTestObject();
+
+        // create branch
+        $branchData = $branchesApi->createBranch($branchName, $branchDescription);
+
+        // create workspace
+        $workspaceBranchApi = new Workspaces($this->getBranchAwareDefaultClient($branchData['id']));
+        $workspace = $workspaceBranchApi->createWorkspace();
+        $connection = $workspace['connection'];
+
+        // delete branch and it should also delete the workspace
+        $branchesApi->deleteBranch($branchData['id']);
+
+        // there is event for deleted branch
+        $this->findLastEvent($this->_client, [
```

Comment:
to ze sa vyhodi eventa pri delete branch uz testujeme `\Keboola\Test\Common\DevBranchesTest::testBranchCreateAndDelete` potom hned testujes ze ta branch tam nieje a wokrspace neexistuje tak to je asi dostacujuce ale kludne to tu nechaj